search/includes/classes/class-healthjob.php

```diff
@@ -115,6 +124,42 @@ public function filter_cron_schedules( $schedule ) {
 		return $schedule;
 	}
 
+	/**
+	 * Check index health
+	 */
+	public function validate_contents() {
+		// Check if job has been disabled
+		if ( ! $this->is_enabled() ) {
+			$this->disable_job();
+
+			return;
+		}
+
+		// Don't run the checks if the index is not built.
+		if ( \ElasticPress\Utils\is_indexing() || ! \ElasticPress\Utils\get_last_sync() ) {
+			return;
+		}
+
+		$results = $this->health->validate_index_posts_content(
+			1,
+			null,
+			null,
+			null,
+			true, // silent
+			false,
+			false,
```

Comment:
😬 the number of args to this function is too high...I like that there's a comment on one, do you think we should comment all of them here?

Reply:
Good point. I switched the signature to use assoc array instead.